Description: gconf -> mateconf, gnome-session -> mate-session
 A patch to ensure compatability with the MATE desktop environment
Author: Jasmine Hassan <jasmine.aura@gmail.com>
Last-Update: 2010-10-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -158,29 +158,29 @@ if test "$use_jpeg" = yes; then
 fi
 
 AC_ARG_ENABLE(schemas,
-  [  --enable-schemas         Build gconf schemas],
+  [  --enable-schemas         Build mateconf schemas],
   [use_schemas=$enableval], [use_schemas=no])
 
 if test x"$use_schemas" = x"yes"; then
-  AC_PATH_PROG(GCONFTOOL, gconftool-2, no)
-  PKG_CHECK_MODULES(SCHEMAS, compiz-gconf, [use_schemas=yes], [use_schemas=no])
-  if test x"$GCONFTOOL" = xno; then
-    AC_MSG_ERROR([gconftool-2 executable not found in your path - should be installed with GConf])
+  AC_PATH_PROG(MATECONFTOOL, mateconftool-2, no)
+  PKG_CHECK_MODULES(SCHEMAS, compiz-mateconf, [use_schemas=yes], [use_schemas=no])
+  if test x"$MATECONFTOOL" = xno; then
+    AC_MSG_ERROR([mateconftool-2 executable not found in your path - should be installed with MateConf])
   fi
   AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
   if test x"$PKG_CONFIG" = xno ; then
     AC_MSG_ERROR([pkg-config executable not found in your path, unable to determine schema directory])
   fi
-  xsltdir=`$PKG_CONFIG --variable=xsltdir compiz-gconf`
+  xsltdir=`$PKG_CONFIG --variable=xsltdir compiz-mateconf`
   AC_SUBST(xsltdir)
 fi
 
-AM_GCONF_SOURCE_2
+AM_MATECONF_SOURCE_2
 
 
 AM_CONDITIONAL(USE_SCHEMAS, test "x$use_schemas" = "xyes")
 if test "$use_schemas" = yes; then
-  AC_DEFINE(USE_SCHEMAS, 1, [Build gconf schemas])
+  AC_DEFINE(USE_SCHEMAS, 1, [Build mateconf schemas])
 fi
 
 AC_OUTPUT([
@@ -191,7 +191,7 @@ data/Makefile
 data/filters/Makefile
 images/Makefile
 images/Oxygen/Makefile
-images/Gnome/Makefile
+images/Mate/Makefile
 Makefile
 metadata/Makefile
 include/Makefile
--- a/images/Makefile.am
+++ b/images/Makefile.am
@@ -1 +1 @@
-SUBDIRS = Oxygen Gnome
\ No newline at end of file
+SUBDIRS = Oxygen Mate
\ No newline at end of file
--- a/metadata/animation.xml.in
+++ b/metadata/animation.xml.in
@@ -59,7 +59,7 @@
 	    <_long>The windows that will be animated.</_long>
 	    <type>match</type>
 	    <default>
-	      <value>(type=Normal | Dialog | ModalDialog | Unknown) &amp; !(name=gnome-screensaver)</value>
+	      <value>(type=Normal | Dialog | ModalDialog | Unknown) &amp; !(name=mate-screensaver)</value>
 	      <value>(type=Menu | PopupMenu | DropdownMenu)</value>
 	      <value>(type=Tooltip | Notification | Utility) &amp; !(name=compiz) &amp; !(title=notify-osd)</value>
 	    </default>
@@ -125,7 +125,7 @@
 	    <_long>The windows that will be animated.</_long>
 	    <type>match</type>
 	    <default>
-	      <value>(type=Normal | Dialog | ModalDialog | Unknown) &amp; !(name=gnome-screensaver)</value>
+	      <value>(type=Normal | Dialog | ModalDialog | Unknown) &amp; !(name=mate-screensaver)</value>
 	      <value>(type=Menu | PopupMenu | DropdownMenu)</value>
 	      <value>(type=Tooltip | Notification | Utility) &amp; !(name=compiz) &amp; !(title=notify-osd)</value>
 	    </default>
--- a/metadata/mag.xml.in
+++ b/metadata/mag.xml.in
@@ -123,13 +123,13 @@
 						<_short>Magnifier image</_short>
 						<_long>Magnifier image.</_long>
 						<hints>file;image;</hints>
-						<default>Gnome/overlay.png</default>
+						<default>Mate/overlay.png</default>
 					</option>
 					<option name="mask" type="string">
 						<_short>Clip mask</_short>
 						<_long>Clip mask.</_long>
 						<hints>file;image;</hints>
-						<default>Gnome/mask.png</default>
+						<default>Mate/mask.png</default>
 					</option>
 					<option name="x_offset" type="int">
 						<_short>X offset of the cursor</_short>
--- a/metadata/Makefile.am
+++ b/metadata/Makefile.am
@@ -53,17 +53,17 @@ xml_DATA     = $(xml_files)
 @INTLTOOL_XML_RULE@
 
 if USE_SCHEMAS
-schemadir   = $(GCONF_SCHEMA_FILE_DIR)
+schemadir   = $(MATECONF_SCHEMA_FILE_DIR)
 schema_files = $(patsubst %.xml.in,compiz-%.schemas,$(xml_in_files))
 schema_DATA  = $(schema_files)
 
 %.schemas: $(xml_files)
 	xsltproc -o $@ $(xsltdir)/schemas.xslt $(subst compiz-,,$*).xml;
 
-if GCONF_SCHEMAS_INSTALL
+if MATECONF_SCHEMAS_INSTALL
 install-data-local:
 	if test -z "$(DESTDIR)" ; then \
-               GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(schema_DATA); \
+               MATECONF_CONFIG_SOURCE=$(MATECONF_SCHEMA_CONFIG_SOURCE) $(MATECONFTOOL) --makefile-install-rule $(schema_DATA); \
        fi
 endif
 endif
--- a/metadata/resizeinfo.xml.in
+++ b/metadata/resizeinfo.xml.in
@@ -2,7 +2,7 @@
 <compiz>
   <plugin name="resizeinfo" useBcop="true">
     <_short>Resize Info</_short>
-    <_long>Display info on resize similar to metacity</_long>
+    <_long>Display info on resize similar to marco</_long>
 	<category>Utility</category>
 	<deps>
 		<relation type="after">
--- a/metadata/workarounds.xml.in
+++ b/metadata/workarounds.xml.in
@@ -6,7 +6,7 @@
         <deps>
         </deps>
         <_short>Workarounds</_short>
-        <_long>Metacity-like workarounds.</_long>
+        <_long>Marco-like workarounds.</_long>
         <display>
             <option type="bool" name="legacy_fullscreen">
                 <_short>Legacy Fullscreen Support</_short>
--- a/src/animation/animation.c
+++ b/src/animation/animation.c
@@ -4607,7 +4607,7 @@ static Bool animInitDisplay(CompPlugin *
     matchInit (&ad->neverAnimateMatch);
     matchAddExp (&ad->neverAnimateMatch, 0, "title=gksu");
     matchAddExp (&ad->neverAnimateMatch, 0, "title=x-session-manager");
-    matchAddExp (&ad->neverAnimateMatch, 0, "title=gnome-session");
+    matchAddExp (&ad->neverAnimateMatch, 0, "title=mate-session");
     matchUpdate (d, &ad->neverAnimateMatch);
 
     WRAP(ad, d, handleEvent, animHandleEvent);
--- a/src/resizeinfo/resizeinfo.c
+++ b/src/resizeinfo/resizeinfo.c
@@ -1,6 +1,6 @@
 /**
  *
- * Compiz metacity like info during resize
+ * Compiz marco like info during resize
  *
  * resizeinfo.c
  *
--- a/src/workarounds/workarounds.c
+++ b/src/workarounds/workarounds.c
@@ -17,7 +17,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  *
- * This plug-in for Metacity-like workarounds.
+ * This plug-in for Marco-like workarounds.
  */
 
 #include <string.h>
--- /dev/null
+++ b/images/Mate/Makefile.am
@@ -0,0 +1,8 @@
+imagesdir = $(imagedir)/Mate
+images_DATA =	       \
+	overlay.png \
+	mask.png
+
+EXTRA_DIST =	       \
+	overlay.png \
+	mask.png
